packages/react-instantsearch-hooks/src/lib/warn.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-console, no-empty */
```

Comment:
I moved these ESLint ignore rules at the top of the file to decluster our "Welcome to debugging" message.